service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/dataset/CloneControlledGcpBigQueryDatasetResourceFlight.java

```diff
@@ -70,9 +73,25 @@ public CloneControlledGcpBigQueryDatasetResourceFlight(
         RetryRules.shortExponential());
     if (mergePolicies) {
       addStep(
-          new ClonePolicyAttributesStep(
+          new MergePolicyAttributesDryRunStep(
```

Comment:
Could this step generation be factored out and reused across controlled resource clones?
Might be more trouble than its worth, but it will be a burden to visit duplicate places (if they are in fact duplicate...)

Reply:
I wondered about that. I started going down that path but it felt like you mentioned - that it was more trouble than it was worth. If you feel strongly about it, I can look into factoring it out again.